textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/connector/OneToNBroadcastConnector.java

```diff
@@ -17,14 +17,18 @@
  */
 public class OneToNBroadcastConnector implements IConnector {
     
+    private static final int PRE_OPEN = -2;
```

Comment:
Add a comment to the codebase to explain the transition of the status using these values.